### PR TITLE
updated demo notebooks

### PR DESCRIPTION
**Pull request by sackh**
Signed-off-by: Kharude, Sachin <sachin.kharude@here.com>

### CODE REVIEW

**Review by omanges — APPROVED** [review]

File: docs/notebooks/spaces_class_example.ipynb [hunk 1]
```diff
@@ -81,9 +104,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Add some data to it\n",
+    "# Countries Polygons/Multiploygons\n",
```
Comment (deeplook) [inline docs/notebooks/spaces_class_example.ipynb:107]: Typo

File: docs/notebooks/xyz_pro_features_examples.ipynb [hunk 2]
```diff
@@ -192,7 +192,7 @@
     "# create two spaces with duplicate data \n",
     "\n",
     "title1 = \"Testing xyzspaces\"\n",
-    "description1 = \"Temporary space containing countries data.\"\n",
+    "description1 = \"Temporary space containing chicago parks data.\"\n",
```
Comment (deeplook) [inline docs/notebooks/xyz_pro_features_examples.ipynb:195]: Chicago

File: docs/notebooks/xyz_pro_features_examples.ipynb [hunk 3]
```diff
@@ -203,7 +203,7 @@
     "space_id1 = space1.info[\"id\"]\n",
     "\n",
     "title2 = \"Testing xyzspaces duplicate\"\n",
-    "description2 = \"Temporary space containing countries data duplicate\"\n",
+    "description2 = \"Temporary space containing chicago parks data duplicate\"\n",
```
Comment (deeplook) [inline docs/notebooks/xyz_pro_features_examples.ipynb:206]: Chicago

File: docs/notebooks/xyz_pro_features_examples.ipynb [hunk 4]
```diff
@@ -299,7 +299,7 @@
     "# create two spaces with duplicate data \n",
     "\n",
     "title1 = \"Testing xyzspaces\"\n",
-    "description1 = \"Temporary space containing countries data.\"\n",
+    "description1 = \"Temporary space containing chicago parks data.\"\n",
```
Comment (deeplook) [inline docs/notebooks/xyz_pro_features_examples.ipynb:302]: Chicago

File: docs/notebooks/xyz_pro_features_examples.ipynb [hunk 5]
```diff
@@ -310,7 +310,7 @@
     "space_id1 = space1.info[\"id\"]\n",
     "\n",
     "title2 = \"Testing xyzspaces duplicate\"\n",
-    "description2 = \"Temporary space containing countries data duplicate\"\n",
+    "description2 = \"Temporary space containing chicago parks data duplicate\"\n",
```
Comment (deeplook) [inline docs/notebooks/xyz_pro_features_examples.ipynb:313]: Chicago

**Review by deeplook — COMMENTED** [review]
Typo plus capitalize "chicago".

**Pull request merged**